website/docs/cloud-docs/users-teams-organizations/teams.mdx

```diff
@@ -29,7 +29,7 @@ In addition to the HCP Terraform UI, you can use the following methods to manage
 
 - [Teams API](/terraform/cloud-docs/api-docs/teams) to list, create, update, and delete teams
 - [Team Members API](/terraform/cloud-docs/api-docs/team-members) to add and delete users from teams
-- [Team Tokens API](/terraform/cloud-docs/api-docs/team-tokens) to generate and delete tokens
+- [Team Tokens API](/terraform/cloud-docs/api-docs/team-tokens) to list, generate, and delete tokens
```

Comment:
```suggestion
- [Team Tokens API](/terraform/cloud-docs/api-docs/team-tokens) to generate and delete tokens and list an organization's team tokens
```

Reply:
Reopening because I think we need to separate `generate` and `delete` from `list` because the scope is different here, let me know if you disagree!